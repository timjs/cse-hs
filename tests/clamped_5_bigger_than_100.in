5
cwgt(dpsv(anxn,anzk),dmhx(aadc(bzpl(ajsq,acrs(btjd,emkt)),bkse(aayo(bsxz(ahmw(alvf,dcmj(clon,cuyr)),wwc),buxl),eobq(bwsd(bhmh,pba),ajkj(cfqa,enqs(ejlo,brix(aofh(iao,dqqj(dxoq,btob)),ayqy)))))),cnaf))
cjcx(efos(ekqa(ojd(cvpk,dgxv(ducd,eqdk)),dkzg(afmn,edbh(cozz(daup(cfba,bndk),dogk),apkx(dfvu(dwia(cvny(dhfl(bflk,cwuf(abdf,dvdm)),dvbs(adzi(crwd,ambr),cpwa)),xxu),bodx),bxst)))),bifd),mjz(bhaz,ekwg))
f(f,b(a(mult,mult(mult(x(mult,x),mult),f(a(plus(b(b,minus),baz),a),mult(f(y(minus(f(plus(baz(b,f(mult,pow)),plus(x,plus(x,mult(a,mult)))),f),pow(minus,b)),x),a),a(a(a(mult,b),y),minus))))),f(pow,f)))
minus(mult,mult(baz(y,y),pow(x(x(minus(y,baz),x(x(minus(mult(mult(y(b,b),minus(mult(x(b,b(y,f)),pow),plus(b(pow(b(baz,y),minus),pow),f))),y),plus),f),baz(minus(pow,minus),b(y,pow(baz,a))))),baz),b)))
adoc(bvxw(dvyz,aopq),uvo(dzfr,dkwl(ekdp(dsrr(ajxx,axzc(eiey,osw(bzqt,dlbn(ewh,dbve)))),cjrd),aetp(bvrx,doaj(elhw(bhef(bmow,bxsn),eohf(cjcc,egve(agbw,egob(absk,ejjj(airz,eisa(aocr,edrn)))))),dvso)))))
