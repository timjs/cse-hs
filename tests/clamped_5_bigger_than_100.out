cwgt(dpsv(anxn,anzk),dmhx(aadc(bzpl(ajsq,acrs(btjd,emkt)),bkse(aayo(bsxz(ahmw(alvf,dcmj(clon,cuyr)),wwc),buxl),eobq(bwsd(bhmh,pba),ajkj(cfqa,enqs(ejlo,brix(aofh(iao,dqqj(dxoq,btob)),ayqy)))))),cnaf))
cjcx(efos(ekqa(ojd(cvpk,dgxv(ducd,eqdk)),dkzg(afmn,edbh(cozz(daup(cfba,bndk),dogk),apkx(dfvu(dwia(cvny(dhfl(bflk,cwuf(abdf,dvdm)),dvbs(adzi(crwd,ambr),cpwa)),xxu),bodx),bxst)))),bifd),mjz(bhaz,ekwg))
f(f,b(a(mult,mult(mult(x(5,x),5),f(a(plus(b(b,minus),baz),a),mult(f(y(minus(f(plus(baz(14,f(5,pow)),plus(9,plus(9,mult(17,5)))),2),pow(15,14)),9),17),a(a(a(5,14),y),15))))),f(26,2)))
minus(mult,mult(baz(y,5),pow(x(x(minus(5,baz),x(x(minus(mult(mult(y(b,17),minus(mult(x(17,b(5,f)),pow),plus(b(pow(b(10,5),minus),23),22))),5),plus),22),baz(minus(23,28),b(5,pow(10,a))))),10),17)))
adoc(bvxw(dvyz,aopq),uvo(dzfr,dkwl(ekdp(dsrr(ajxx,axzc(eiey,osw(bzqt,dlbn(ewh,dbve)))),cjrd),aetp(bvrx,doaj(elhw(bhef(bmow,bxsn),eohf(cjcc,egve(agbw,egob(absk,ejjj(airz,eisa(aocr,edrn)))))),dvso)))))
